targets/mega65/vic4.c

```diff
@@ -243,18 +243,20 @@ static void vic4_update_sideborder_dimensions ( void )
 	if (REG_CSEL) {	// 40-columns?
 		border_x_left = FRAME_H_FRONT + SINGLE_SIDE_BORDER;
 		if (!REG_H640)
-			border_x_right = FRAME_H_FRONT + TEXTURE_WIDTH - SINGLE_SIDE_BORDER - 1;
```

Comment:
Here for example, your PR want to rename TEXTURE_WIDTH back to SCREEN_WIDTH, though in theory you should have for now the state where SCREEN_WIDTH (and HEIGHT) is renamed to TEXTURE...

Reply:
Uhmmm... i dont even remember touching that...

Reply:
I'm a bit clueless, to be honest :( OK, maybe it was not a good idea from me to merge my changes first to you, but it still does not explain, how things changed back to SCREEN_ ... when I have no such string in the whole mega65 subtree any more ...

Reply:
Something got mixed, in my uncomitted vic4.h there were SCREEN_WIDTH and TEXTURE_WIDTH both present. Now I have the synced vic4.h with only TEXTURE_WIDTH present, changed the only 3 occurrences in vic4.c and all it's going well.  Go figure what happened.